ERCS/erc-76xx.md

```diff
@@ -0,0 +1,142 @@
+---
+eip: 7600
```

Comment:
```suggestion
eip: 7627
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.